primitives/src/policy.rs

```diff
@@ -58,6 +58,12 @@ impl Policy {
     /// The maximum allowed size, in bytes, for a micro block body.
     pub const MAX_SIZE_MICRO_BODY: usize = 100_000;
 
+    /// Maximum lengths for transaction sender_data and recipient_data fields used during deserialization.
+    pub const MAX_TX_SENDER_DATA_SIZE: usize = 2;
```

Comment:
Where are 2 bytes used? I think only txs from the staking contract use this and they use 1 byte only. Or does this include the 1 byte "prefix" for the data length?

Reply:
Nah, it is just one byte you are correct.